cypress/integration/pages/details.spec.js

```diff
@@ -127,6 +127,16 @@ describe('Details Page', () => {
       cy.visit('http://localhost:3001/customers/1/view');
       cy.get(vulnerabilities).should('not.exist');
       cy.get(assets).should('not.exist');
-    })
+    });
+
+    it('redirects to the correct url after creating a new snapshot', () => {
+      cy.visit('http://localhost:3001/customers/10/view');
+      cy.get(vulnerabilities).should('exist');
+
+      cy.get('[data-testid=add-vulnerability-snapshot-button]').click({
+        force: true
+      });
+      cy.location('pathname').should('eq', '/snapshots/10');
```

Comment:
Can we test the whole location here rather than just the pathname?

Reply:
So the path name would be _http://localhost:3001/snapshots/10_ and that's why those env vars are set in circle. The reason it's the same port as UI is because cypress can't redirect to diff origin and cy.route() is stubbing out the backend in general, didn't manage to do it on cross origin, so it would still be pointing to 3001.
Here, not sure if it's worth checking the full path, I guess it would sort of check that it gets the correct env var, so I can change this